Data/Input/default.xci

```diff
@@ -594,23 +594,23 @@ mode=Config2
 type=key
 data=APP3
 event=Mode Config3
-label=Config\n2/3
+label=Config\nPage 2/3
 location=3
 
 mode=Config2
 type=key
 data=6
 event=Setup Profile
 event=Mode default
-label=Profiles
+label=Profile\nManager
 location=5
 
 mode=Config2
 type=key
 data=8
 event=WaypointEditor
 event=Mode default
-label=Waypoints Editor
+label=Waypoint\nEditor
```

Comment:
I don't like having explicit layout decisions in translatable strings. We should separate content and layout as much as possible. If the button doesn't line-break automatically, we should improve that algorithm instead of adding layout specs here.